SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -109,27 +96,8 @@ uint32 GetWorkerCountFromWorldSettings(const UWorld& World, bool bForceNonEditor
 		->GetMinimumRequiredWorkerCount();
 }
 
-bool FillWorkerConfigurationFromCurrentMap(FWorkerTypeLaunchSection& OutWorker, FIntPoint& OutWorldDimensions)
-{
-	if (GEditor == nullptr || GEditor->GetWorldContexts().Num() == 0)
-	{
-		return false;
-	}
-
-	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
-	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
-
-	UWorld* EditorWorld = GEditor->GetEditorWorldContext().World();
-	check(EditorWorld != nullptr);
-
-	OutWorker = SpatialGDKEditorSettings->LaunchConfigDesc.ServerWorkerConfig;
-	OutWorker.NumEditorInstances = GetWorkerCountFromWorldSettings(*EditorWorld);
-
-	return true;
-}
-
 bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchConfigDescription* InLaunchConfigDescription,
```

Comment:
minor: prefer a (scoped) enum over bool in these situations (non obvious from signature what bool indicates).